builder/triton/driver_triton.go

```diff
@@ -193,3 +225,29 @@ func waitFor(f func() (bool, error), every, timeout time.Duration) error {
 
 	return errors.New("Timed out while waiting for resource change")
 }
+
+func mostRecentImages(images []*compute.Image) *compute.Image {
+	return sortImages(images)[0]
+}
+
+type imageSort []*compute.Image
+
+func sortImages(images []*compute.Image) []*compute.Image {
+	sortedImages := images
+	sort.Sort(sort.Reverse(imageSort(sortedImages)))
```

Comment:
I'm not sure I understand the `sort.Sort(sort.Reverse(...))` here.  Why not just `sort.Sort(imageSort(sortedImages))`?

Reply:
@sean- this has been picked up *exactly* from the terraform triton and AWS providers

I must say I cannot remember why I did it - so I can look into this and follow up on it

Reply:
ok  @sean- I just removed and retested this. It doesn't work correctly without the reverse

With the reverse:

```
packer build \
        -var "triton_account=stack72_joyent" \
        -var "triton_key_id=40:9d:d3:f9:0b:86:62:48:f4:2e:a5:8e:43:00:2a:9b" \
        -var "triton_key_material=""" \
        -var "image_version=1.0.0" \
        new-template.json
triton output will be in this color.

==> triton: Selecting an image based on search criteria
==> triton: Based, on given search criteria, Machine ID is: "7b5981c4-1889-11e7-b4c5-3f3bdfc9b88b"
^C==> triton: Waiting for source machine to become available...
```

Without the reverse
```
packer build \
        -var "triton_account=stack72_joyent" \
        -var "triton_key_id=40:9d:d3:f9:0b:86:62:48:f4:2e:a5:8e:43:00:2a:9b" \
        -var "triton_key_material=""" \
        -var "image_version=1.0.0" \
        new-template.json
triton output will be in this color.

==> triton: Selecting an image based on search criteria
==> triton: Based, on given search criteria, Machine ID is: "49b22aec-0c8a-11e6-8807-a3eb4db576ba"
^C==> triton: Waiting for source machine to become available...
```

Only the first is correct - the second Machine ID isn't

Reply:
Huh, interesting.  Let's pull that apart later.  Thanks for looking into this!

Reply:
This probably warrants further investigation at some point in future, but since every other source is working like this I think that can be deferred since it is working now.